suites/squid/rbd/tier-2_rbd_group.yaml

```diff
@@ -164,3 +164,20 @@ tests:
       module: test_rbd_clone_flatten_group_snapshot.py
       name: Test cloning and flattening group snapshot
       polarion-id: CEPH-83594299
+
+  - test:
+      desc: Verify group snapshot rollback not allowed if memberships dont match
+      config:
+        group: test_group
+        snap: test_snap
+        rep_pool_config:
+          num_pools: 2
+          num_images: 1
+        ec_pool_config:
+          num_pools: 2
+          num_images: 1
+        fio:
+          size: 10M
+      module: test_rbd_disallow_group_snapshot_rollback.py
+      name: Disallow Group Snapshot Rollback if Memberships Don't Match
```

Comment:
make it as `Dis-allow`

Reply:
Test name has been maintained the same as that of in polarion. https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83594544. 
Let me know if the test name be changed in polarion as well.